src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -575,7 +575,23 @@ public static IEnumerable<object[]> GetTestProcess()
             yield return new object[] { currentProcess, Process.GetProcessesByName(currentProcess.ProcessName, "127.0.0.1").Where(p => p.Id == currentProcess.Id).Single() };
         }
 
-        [Theory, PlatformSpecific(PlatformID.Windows)]
+        private static bool ProcessPeformanceCounterEnabled()
+        {
+            try
+            {
+                int? value = (int?)Microsoft.Win32.Registry.GetValue(@"HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\PerfProc\Performance", "Disable Performance Counters", null);
```

Comment:
I assumed that the since the only test that uses this condition is marked as Windows only via the PlatformSpecific attribute, the condition would not be run on UNIX.

Will the condition be evaluated even though the test is marked as PlatformSpecific?  If so, I could add guards in here.